src/tool/python/code_writers.h

```diff
@@ -12,6 +12,41 @@ namespace xlang
         w.write("@ = __ns__.@\n", type.TypeName(), type.TypeName());
     }
 
+    void write_python_enum_name(writer& w, std::string_view const& name)
+    {
+        // None is a python keyword, so project it as NONE
+        if (name == "None")
+        {
+            for(auto&& c : name)
```

Comment:
why not just write("NONE")

Reply:
in case I need to add other keywords down the road